src/main/java/com/google/sps/data/Room.java

```diff
@@ -35,11 +35,6 @@
     private Queue<Video> videos;
     private Long roomId;
 
-    //Room factory function
-    public static Room createRoom(List<Member> members, Queue<Video> videos, Queue<Message> messages){
-        return new Room(members, videos, messages, null);
-    }
-
     //Creates a room object from a Datastore Key
     public static Room fromRoomId(long roomId) {
```

Comment:
Is the  fromRoomKey private helpful function needed or can you move its impl directly to fromRoomId?

Reply:
It is just to separate the logic into smaller functions.

Reply:
I failed to see how does this helpful function split the logic. In current version, fromRoomId is just a one line function and all logic simply moves to fromRoomKey.

Reply:
Ok, I'll change it